Engineering/CodeReviews/Go.md

```diff
@@ -18,6 +18,44 @@ All developers should run ```gofmt``` in a pre-commit hook to ensure standard fo
 
 ```go vet``` is a static analysis tool that checks for common go errors, such as incorrect use of range loop variables or misaligned printf arguments. [CSE](../CSE.md) Go code should be able to build with no ```go vet``` errors.
 
+### ```gometalinter```
+
+```gometalinter``` is a tool that concurrently runs numerous linters and normalises their output to a standard format.
+In order to automate the linting process, you can include a `gometalinter.json` settings file in your project and run the tool with
+```sh 
+gometalinter --config ./gometalinter.json ./...
+```
+
+`gometalinter.json`:
+```json
+{
+    "Vendor": true,
+    "Deadline": "5m",
+    "Sort": ["linter", "severity", "path", "line"],
+    "EnableGC": true,
+    "WarnUnmatchedDirective": true,
+    "DisableAll": true,
+    "Linters": {
+        "goimports": {
+            "Command": "goimports -l --local github.com/{your-org}/{your-repo}"
+        }
+    },
+    "Enable": [
+        "deadcode",
+        "gofmt",
+        "gocyclo",
+        "goimports",
+        "golint",
```

Comment:
using `golint` via `gometalinter` contradicts the statement made on line 61. Personally, I **do** run `golint` as part of my CI builds on a number of projects and use `//nolint: golint` to disable false positives. What are your thoughts? remove line 61 or remove `golint` from the `gometalint` linters list?

Reply:
Good catch. I think adding `//nolint: golint` is better than removing it completely. It forces developers to "know" their code.

Reply:
@timfpark

Reply:
There will always be false positives in any major project.  My vote would to be go with adding the //nolint: golint guidance to work around it when it happens but have the default be to lint the project.

Reply:
@timfpark updated https://github.com/Microsoft/code-with-engineering-playbook/pull/12/commits/74326a035115edf4f5f3a03e722ac603ffffa507